landscape.yml

```diff
@@ -3506,6 +3506,12 @@ landscape:
             logo: ./hosted_logos/scalyr.svg
             twitter: 'https://twitter.com/scalyr'
             crunchbase: 'https://www.crunchbase.com/organization/scalyr'
+          - item:
+            name: LogDNA
+            homepage_url: 'https://www.logdna.com'
+            logo: ./hosted_logos/logdna.svg
```

Comment:
Your logo needs to include the product name. Please change this to:
```yaml
            logo: 'https://www.cncf.io/wp-content/uploads/2017/09/logdna.svg'
```

Reply:
@dankohn, yea, I have noticed - working on it, thanks!

Reply:
@dankohn, updated, lemme know whether there is another issue :)